designs/virtualization/4.x/create-vm-dropdowns/create-vm-dropdowns.md

```diff
@@ -0,0 +1,137 @@
+---
+parent: Virtualization
+version: 4.x
+---
+
+# create VM dropdowns
```

Comment:
This title should be updated now b/c this update covers more than just the drop downs